app/src/test/java/org/mozilla/fenix/ext/NotificationManagerCompatTest.kt

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.ext
+
+import androidx.core.app.NotificationChannelCompat
+import androidx.core.app.NotificationManagerCompat
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.fenix.helpers.FenixRobolectricTestRunner
+
+@RunWith(FenixRobolectricTestRunner::class)
+class NotificationManagerCompatTest {
+
+    private lateinit var tested: NotificationManagerCompat
+
+    @Before
+    fun setup() {
+        tested = mockk(relaxed = true)
+
+        mockkStatic("org.mozilla.fenix.ext.NotificationManagerCompatKt")
+    }
+
+    @Test
+    fun `areNotificationsEnabledSafe returns false when areNotificationsEnabled throws an exception`() {
```

Comment:
Nice tests 👍 
Generally I think the [GIVEN, WHEN, THEN] or [WHEN, THEN] format is preferred for test names, examples: https://github.com/mozilla-mobile/fenix/blob/main/app/src/test/java/org/mozilla/fenix/gleanplumb/DefaultMessageControllerTest.kt & https://github.com/mozilla-mobile/fenix/blob/main/app/src/test/java/org/mozilla/fenix/browser/BrowserFragmentTest.kt

Reply:
Thanks for the reference 👌  updated to follow the same pattern